### PR TITLE
Adds Aguilar, Aion, Akai, Behringer, BOSS, Donner, Electro-Harmonix, FTelettronica, Tech21, The King of Gear, Walrus, and pedal requests

### LINKED ISSUE #1146: Pedal Request: Nunez Amps Tetra-Fet Drive
Brand: Nuenez Amps
Pedal: Tetra-Fet Drive 
Size: 2.59 x 4.94

Screenshot_20190104-103027_Instagram

### LINKED ISSUE #1179: Pedal request fix: Dunlop Cry Baby Q Zone
Submitted a request last week. The ticket was closed but the issue wasn't resolved. Thanks. 
3E8FF930-6899-4CB7-A477-2E4FC5358BE2

### LINKED ISSUE #1181: Pedal Request: Oto Machines Bim, Bam, and Boum
OTO_BOUM_TRANSPARENT
Boum: 5.71" x 5.92"

OTO_BAM_TRANSPARENT
Bam: 5.71" x 5.92"

OTO_BIM_TRANSPARENT
Bim: 5.71" x 5.92"

### LINKED ISSUE #1189: GCB Wahs
What the heck happened to all the Dunlop GCB Wahs ??

### LINKED ISSUE #1192: Garage Tone Drive Train
Thank you very much!
Dimensions
12cm 
8cm

DrivetrainOD

### LINKED ISSUE #1193: Cry Baby GCB65 Custom Badass Dual Inductor Wah
814TDOBwJYL _AC_SX679_-removebg-preview
Height: 10.0 inches
Width: 4.3 inches

### LINKED ISSUE #1197: [Reopened] Friedman Gold-72 No More Tears Wah Pedal
Reopening this as the previous issue #740 has been closed but the actual image on pedalplayground.com is sill incorrect as per the screenshot below (notice how the proportion is incorrect, with the image being vertically stretched):

### LINKED ISSUE #1198: Pedal Request : Radial Engineering AC-Driver
image

https://www.radialeng.com/product/ac-driver

### LINKED ISSUE #1200: Pedal Request: Seymour Duncan Vapour Trails Deluxe
Brand: Seymour Duncan
Pedal: Vapour Trails Deluxe
Size: 3.57” x 4.80"

SS

### LINKED ISSUE #1201: Pedal Request: Caline Pure Sky
Trimmed image (/w transparency; png) (not saved for web)
caline_pure_sky

Dimensions:
4.5 in x 2.70 in

### LINKED ISSUE #1209: Pedal request: Lauren Audio Mustang
Lauren Audio Mustang 

w: 3.5
h: 5.32

4A24FBEA-514A-460D-9FFA-17AA29A20F36

### LINKED ISSUE #1217: Pedal Request: Boss PQ-4 Parametric Equalizer
bosspq4

### LINKED ISSUE #1222: Pedal Request: MJM London Fuzz (Small, Silver)
width 2.71
height 4.41
mjm-london-fuzz-silver

### LINKED ISSUE #1223: Pedal Request: MJM Phantom Overdrive
width 2.91
height 4.80
mjm-phantom-overdrive

### LINKED ISSUE #1224: Pedal Request: Cusack Tap-A-Whirl (Red)
width 4.68
height 4.13
cusack-tap-a-whirl-v1

### LINKED ISSUE #1239: Pedal Request: VFE Klein Bottle 2022
| Pedal | Width | Depth | Height | Image |
|-------|:-----:|:-----:|:------:|:-----:|
|   **_VFE_ <br> Klein Bottle 2022**    |   5.625"    |   4.63"    |  1.5"    |       |

### LINKED ISSUE #1242: Pedal Request: Deadbug Circuits 1968
width: 2.5
height: 4.75
deadbug-circuits-1968

### LINKED ISSUE #1246: Pedal Request: 3rd Power Roosevelt Drive
https://3rdpower.com/roosevelt-drive/

image

### LINKED ISSUE #1262: Rocktron Midi Raider
How about adding a Rocktron Midi Raider to the database. Hopefully this image is acceptable:
Midi-Raider

### LINKED ISSUE #1267: Pedal Request: Radial BigShot ABY
| | |
|--|--|
| Pedal   | **Radial BigShot ABY** |
| Brand   | Radial Engineering        |
| Dimensions (W x D x H) | 3.6” x 4” x 2” (92mm x 100mm x 51mm)          |
| Image | BigShot-ABY|
|Transparent Image|BigShot-ABY-transparent|

### LINKED ISSUE #1268: Pedal Request: Boredbrain Terminal v2
| | |
|--|--|
| Pedal   | **Boredbrain Terminal v2** |
| Brand   | Boredbrain        |
| Dimensions (W x D x H) | 2.75" x 4.5" x 2.5" (70mm x 114mm x 64mm)          |
| Image |Boredbrain-Terminal-v2 |
|Transparent Image|Boredbrain-Terminal-v2-transparent|

### LINKED ISSUE #1274: Pedal requests: Hughes and Kettner Tube Factor
Thanks to anyone who can help!

6.299" Width, 8.465" Height 

Tube Factor

### LINKED ISSUE #1275: Pedal Request: Hughes and Kettner Replex
Thanks so much to anyone who can help!

10.079" Width, 8.465" Height

hughes-kettner-replex

### LINKED ISSUE #1286: Boss SL-2 Slicer
SL-2_F_852914-51df52db1d98fe2b8863471716df1192

DIMENSIONS: 2-7/8" W x 5-1/8" D (typical boss compact dimensions)

### LINKED ISSUE #1288: Pedal request: Boss DS-1-4A Anniversary Distortion
A651E89C-ED70-42A4-AFFF-84BE578AB095

### LINKED ISSUE #1310: Pedal Request: TC Helicon Voicelive 3 (not the extreme version)
Dimensions: 8.5 inch x 13.8 inch

VOICELIVE-3_P0CDY_Front_XL

Thank you

### LINKED ISSUE #1311: Pedal Request: NUX NMP-2 Dual Footswitch Controller
Dimensions: 4.6cm x 9.5cm

NUX-NX-NMP2

Thank you

### LINKED ISSUE #1336: Pedal Request : Boss SL-2 Slicer
https://www.boss.info/ca/products/sl-2/

### LINKED ISSUE #1349: Add Ibanez CSL Stereo Chorus
I suggest adding the **Ibanez CLS Stereo Chorus** pedal. It has the dimensions of 3 x 4.9 inches. I prepared the image as per the instructions in the Readme file.
Sem Título-1

The json code would be this...

```js

	{
		"Brand": "Ibanez",
		"Name": "CSL Stereo Chorus",
		"Width": 3,
		"Height": 4.9,
		"Image": "ibanez-csl.png"
	},
```
=D Thanks!

### LINKED ISSUE #1360: Dunlop Mini 535Q Wah disappeared?
Its placeholder loads, with no image.
What gives?

### LINKED ISSUE #1361: Pedal Request : Lauren Audio Mustang
https://www.lauren.audio/

### LINKED ISSUE #1374: Request: LPD Eighty7 (Black)
Brand: LPD
Model: Eighty7 (Black)
Size: 2.68 x 4.5
LPD-1

### LINKED ISSUE #1410: Pedal Request: Latent Lemon Audio - Hurts
Dimensions: 4.72 x 3.70
Latent_Lemon_Audio_Hurts

### LINKED ISSUE #1411: Pedal Request: Latent Lemon Audio - Hurts - Bare Metal
Dimensions: 4.72 x 3.70
Latent_Lemon_Audio_Hurts_BareMetal

### PR DESCRIPTION
**Pull request by XlosVSM**
**Adds**
- 3rd Power Roosevelt Drive
- Aguilar Storm King
- Aion Andromeda
- Aion Cerulean
- Aion Convex
- Aion Equinox
- Aion Graviton
- Aion Halo
- Aion Helios
- Aion Lumin
- Aion Phobos
- Aion Quantum
- Aion Radian
- Aion Refractor
- Aion Rift
- Aion Solaris
- Aion Solstice
- Aion Vector
- Akai E2 Headrush
- Behringer Dual-Phase
- Boredbrain Terminal v2
- BOSS DS-1-4A
- BOSS FS-1-WL
- BOSS OD-2R
- BOSS PQ-4
- BOSS SL-2
- Caline Pure Sky
- Cry Baby GCB65 Custom Badass Dual Inductor Wah
- Cusack Tap-A-Whirl (Red)
- Deadbug Circuits 1968
- Donner Blues Drive
- Donner Soph Gate
- Electro-Harmonix Hot Tubes (Original)
- Electro-Harmonix Tube EQ
- FTelettronica Tuwat Resonant Envelope Filter
- FTelettronica Utopian Fuzz
- FTelettronica Venomous Octopus
- FTelettronica Wave Reflector
- FTelettronica Weird Sound Generator Elite
- FTelettronica Zero Sum Game
- FTelettronica Zeroth Law
- Garage Tone Drive Train
- Hughes and Kettner Tube Factor
- Hughes and Kettner Tube Replex
- Ibanez CSL Stereo Chorus
- Latent Lemon Audio Hurts
- Latent Lemon Audio Hurts (Bare Metal)
- Lauren Audio Mustang
- LPD Eighty7 (Black)
- MJM FX London Fuzz
- MJM FX Phantom Overdrive
- Nuñez Amps Tetra-Fet Drive (2020 CME Pink)
- Oto Machines Bam
- Oto Machines Bim
- Oto Machines Boum
- Radial Engineering AC-Driver
- Radial BigShot ABY
- Rocktron MIDI Raider
- Seymour Duncan Vapour Trails Deluxe
- TC Helicon Voicelive 3
- Tech21 Bass Driver Programmable
- The King of Gear Feral Glitch
- VFE Klein Bottle 2022
- Walrus Ages (Kamakura Series)
- Walrus ARP-87 (Kamakura Series)
- Walrus ARP-87 (Obsidian Series)
- Walrus Eras (Kamakura Series)
- Walrus Eras (Obsidian Series)
- Walrus Eras (Retro Edition)
- Walrus Fathom (Kamakura Series)
- Walrus Julia V2 (Kamakura Series)
- Walrus Julianna (Kamakura Series)
- Walrus Julianna (Obsidian Series)
- Walrus Lore (Kamakura Series)
- Walrus Monument V2 (Kamakura Series)
- Walrus Monument V2 (Obsidian Series)
- Walrus Polychrome (Kamakura Series)
- Walrus Slötvå (Obsidian Series)

**Fixes**
- Donner AB/Y dimensions
- Friedman Gold-72 images (blank space was forgotten to be removed)
- Improves BOSS DS-1w image
- Moves Malekko Ekko 616 MKII FE to the correct position alphabetically
- Moves MXR Bass Innovations to its own section
- Removes all.html, as I think it is pointless
- Removes forgotten Dunlop wahs (if a pedal on your website has gone invisible, delete it and readd it)
- Removes unused images
- Spelling
- Twinote BBD Chorus dimensions
- Various file names
- VFE Klein Bottle 2022 dimensions

closes #1146 (Not the black one but good enough for the moment)
closes #1179
closes #1181
closes #1189
closes #1192
closes #1193
closes #1197
closes #1198
closes #1200
closes #1201 
closes #1209
closes #1217
closes #1222
closes #1223
closes #1224
closes #1239
closes #1242
closes #1246
closes #1262 
closes #1267 
closes #1268 
closes #1274 
closes #1275 
closes #1286 
closes #1288 
closes #1310 
closes #1311 already in
closes #1336 
closes #1349 
closes #1360 
closes #1361 
closes #1374 
closes #1410 
closes #1411

### CODE REVIEW

**Comment by ArislanShiva** [comment]
Hey, just FYI I had the CTC Unobtainium done already.

**Review by ArislanShiva — CHANGES_REQUESTED** [review]
Also, please don't change the dimensions for the Nux switch.  Gotta account for jacks.

**Comment by XlosVSM** [comment]
I am not going to add more to this pr because it is already huge in changes, and I do not want to bother people with more whens when the merge happens. @tehtrav how long before it is merged?

File: public/data/pedals.json [hunk 1]
```diff
@@ -28014,6 +28014,13 @@
 		"Height": 4.37008,
 		"Image": "vfe-blueprint-v4.png"
 	},
+	{
+		"Brand": "VFE",
+		"Name": "Klein Bottle 2022",
+		"Width": 2.59843,
+		"Height": 4.37008,
+		"Image": "vfe-klein-bottle-2022.png"
+	},
```
Comment (lightyrs) [inline public/data/pedals.json:29773]: @XlosVSM Dimensions are off here. This should be:

```json
{
    "brand": "VFE",
    "name": "Klein Bottle 2022",
    "width": 5.625,
    "height": 4.63,
    "image": "vfe-klein-bottle-2022.png"
  }
```

**Comment by tehtrav** [comment]
fix conflicts and I'll pull it in, sorry for the delay

**Comment by XlosVSM** [comment]
@tehtrav fixed

**Review by ArislanShiva — CHANGES_REQUESTED** [review]
Please double check your measurements on these. 

1590BB enclosure is 3.7" wide and 4.7" tall, not counting the jacks.

**Comment by ZYMECORE** [comment]
When will this merge post? This pull is almost 6 months old!

**Comment by ArislanShiva** [comment]
> When will this merge post? This pull is almost 6 months old!

Every time @tehtrav goes to pull this one in there are conflicts that need to be resolved.

If @XlosVSM doesn't stop with the broader file/name edits that are causing the conflicts it's never gonna get pulled in.

**Comment by XlosVSM** [comment]
@tehtrav still waiting for merge

**Comment by oooctave** [comment]
Lauren Audio Mustang is still missing from the database from this push

**Pull request closed without merging**